src/gui/newwizard/pages/oauthcredentialssetupwizardpage.h

```diff
@@ -17,7 +17,7 @@
 #include "abstractsetupwizardpage.h"
 
 namespace Ui {
-class OAuthCredentialsSetupWizardPage;
+class CredentialsSetupWizardPage;
 }
 
 namespace OCC::Wizard {
```

Comment:
For line 25: class doc? E.g.:

This wizard page uses the `OAuthLoginWidget` to get the credentials.

Reply:
Isn't that quite obvious from the name and code?

Reply:
I guess since we have not done this anywhere else, we'll leave it as it is.